src/connectors/toggleRefinement/connectToggleRefinement.js

```diff
@@ -105,10 +105,13 @@ export default function connectToggleRefinement(renderFn, unmountFn = noop) {
     return {
       $$type: 'ais.toggleRefinement',
 
-      getConfiguration() {
-        return {
+      getConfiguration(state) {
+        return state.setQueryParameters({
           disjunctiveFacets: [attribute],
-        };
+          disjunctiveFacetsRefinements: {
+            [attribute]: [],
```

Comment:
We don't use the previous value, it's inconsistent with the other changes currently made. But we won't use it once we merge the lifecycles so it's fine.

Reply:
Not sure what you mean here, why is the Current refinement not taken in account?